File: 프로그래머스/1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.py
def solution(n, arr1, arr2):
    answer = []
    for i,j in zip(arr1, arr2):
        row = str(bin(i|j)[2:])
        row = row.rjust(n, '0')
        row = row.replace('1', '#').replace('0', ' ')
        answer.append(row)
    return answer

File: 프로그래머스/1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.js
function solution(n, arr1, arr2){
  let answer = [];
  const arrs = arr1.map((v, i) => [v, arr2[i]]);
  arrs.forEach(([i, j]) => {
    let row = (i|j).toString(2).padStart(n, 0);
    console.log(row);
    answer.push(row.replace(/1/gi, "#").replace(/0/gi, " "));
  })

  return answer;
}